modules/sdk-coin-apt/src/apt.ts

```diff
@@ -197,4 +199,12 @@ export class Apt extends BaseCoin {
       throw new Error('Failed to rebuild transaction');
     }
   }
+
+  /** @inheritDoc */
+  auditDecryptedKey({ multiSigType, prv, publicKey }: AuditDecryptedKeyParams): void {
+    if (multiSigType !== 'tss') {
+      throw new Error('Unsupported multiSigType');
+    }
+    auditEddsaPrivateKey(prv, publicKey ?? '');
+  }
```

Comment:
Nit: you can create abstract class for ED and EC and have these coins extend that instead of implementing the same function over and over again.